src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_resolver.py

```diff
@@ -168,7 +168,7 @@ def get_names(self, var):
         used___dict__ = False
         try:
             names = dir(var)
-        except TypeError:
+        except Exception:
```

Comment:
Should this do `object.__dir__(var)` as a fallback?

Reply:
The fallback was already there if  (it uses `var.__dict__`, which I guess is roughly the same as what `object.__dir__(var)` does). i.e.: https://github.com/microsoft/ptvsd/blob/e17b3d0c8558fb6ff207704e6804c490eab8e664/src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_resolver.py#L174